build-tools/makefile_components/base_build_python-docker.mak

```diff
@@ -11,7 +11,7 @@ SHELL := /bin/bash
 
 PWD := $(shell pwd)
 
-BUILD_IMAGE ?= drud/golang-build-container:v0.5.2
+BUILD_IMAGE ?= drud/golang-build-container:v0.5.4
```

Comment:
shouldn't this be 0.5.5. too?

Reply:
Really, it should be removed, as the BUILD_IMAGE is used nowhere. It's actually a result of me not catching it in the build-tools PR. 

I think I should do a PR to just remove this from the python-docker, pretty sure it can do no harm.